### DISCUSSION TITLE
GitHub copilot - should i show this to my students?

### DISCUSSION THREAD

**Discussion by jpanzen** [community/community#48446] [GitHub Education]
Hey, just started using copilot and honestly I'm amazed by how easy it is now to create code. I'm having a huge dilemma, i think my students should know about copilot, but I'm worried they'll start to be heavily dependant on it and won't really learn to code in the end. What's your view on this?

**Comment by radar-blip** [comment]
You should teach them to write their own code. 👍🏻
